resources/js/views/global/Supercluster.vue

```diff
@@ -268,7 +268,7 @@ export default {
         litterArtPoints = L.geoJSON(null, {
             pointToLayer: createArtIcon,
             onEachFeature: onEachArtFeature
-        });
+        }).addTo(map);
```

Comment:
this is not wanted. By default, we show the global data. If the user wants to see the litter art data, they have to add the layer